examples/pod_namespace_watch.py

```diff
@@ -29,8 +33,18 @@ def main():
         count -= 1
         if not count:
             w.stop()
+    print("Finished namespace stream.")
 
-    print("Ended.")
+    for event in w.stream(v1.list_pod_for_all_namespaces, timeout_seconds=10):
```

Comment:
the watch pod stream will start only after 10 namespace watch events are observed, which isn't always happening. 

how I would imagine running this example: 
- run the example
- see events get printed (`CREATE kube-system`, `CREATE default`)
- the example hangs
- in another terminal, create/update/delete a few namespaces 
- see a few more namespace watch events get printed (CREATE/UPDATE/DELETE)
- the example hangs since not sufficient (10) events are observed
- ctrl-C to stop the example

Reply:
I updated the comments. It only hangs until 10 events are observed or the `timeout_seconds` threshold is reached, which is set to 10 seconds.

Reply:
ah that's right. Thanks